packages/toolpad-app/src/toolpadDataSources/rest/server.ts

```diff
@@ -136,21 +136,21 @@ async function execBase(
   project: IToolpadProject,
   connection: Maybe<RestConnectionParams>,
   fetchQuery: FetchQuery,
-  params: Record<string, string>,
+  params: BindableAttrValue<any>,
```

Comment:
I think the correct type now here would be something like `Record<string, string | BindableAttrValue<any>>`?